app/models/concerns/searchable.rb

```diff
@@ -9,9 +9,10 @@ module Searchable
     pg_search_scope :search,
                     against: searchable_fields_and_ranks,
                     using: { tsearch: {
-                               any_word: true,
+                               any_word: false,
```

Comment:
Hi Ian, just curious for this change,  this change seems to me will affect other case types, is it any consequence outcome for this?

Reply:
Sorry - missed this! Yes, we want to experiment with it on the London side also as we suspect it may give better results for them too